documentation/staging/content/managing-domains/accessing-the-domain/access-admin-console-em.md

```diff
@@ -0,0 +1,210 @@
+---
+title: "Access the Administration Console"
```

Comment:
"Access the" --> "Use the ..." (this is the same terminology as the other sections in teh same chapter...)

Reply:
Changed.